azure-pipelines.yml

```diff
@@ -312,7 +312,7 @@ jobs:
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: windows
-      osVersion: 10
+      osVersion: RS1
```

Comment:
this is probably an unrelated question but I thought we were going to move these to an RS5 pool of some sort..

Reply:
I was thinking that if we wanted to keep the graphs consistent, we would do that post-release of 3.0, at which point, the machines will get all of the helix updates that allow them to use the artifacts directory.